config/config.go

```diff
@@ -40,6 +43,16 @@ func (c *config) load(filePath string) error {
 		return err
 	}
 
+	c.Hardcoded.Errors = map[string]models.ShortLink{}
+	for _, e := range c.Hardcoded.Error {
+		c.Hardcoded.Errors[e.Key] = e
```

Comment:
What if key is missing or empty?

Reply:
And what if there are duplicates? For sure need to handle those cases